install/kubernetes/cilium/templates/spire/server/configmap.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if and .Values.auth.mTLS.spire.enabled .Values.auth.mTLS.spire.install.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: spire-server
+  namespace: {{ .Values.auth.mTLS.spire.install.namespace }}
+data:
+  server.conf: |
+    server {
+      bind_address = "0.0.0.0"
+      bind_port = "8081"
+      socket_path = "/tmp/spire-server/private/api.sock"
+      trust_domain = {{ .Values.auth.mTLS.spire.trustDomain | quote }}
+      data_dir = "/run/spire/data"
+      log_level = "INFO"
+      #AWS requires the use of RSA.  EC cryptography is not supported
```

Comment:
`AWS requires the use of RSA. EC cryptography is not supported`

This comes out of some default spire thing, but how much does it affect EKS, and mostly cilium as we are the only key consumer? I would way rather drop RSA here.... If we have to keep it we should debate the key size also...

Reply:
updated to rsa-4096, verification was done with fresh installation as well.